.github/workflows/build.yml

```diff
@@ -8,18 +8,21 @@ on:
     branches:
       - "*"
 
+anchors:
```

Comment:
I think this is preventing this job from running, can you try defining the strategy as it was and see if the jobs run?

Reply:
though i'd expect it to trigger & fail...

Reply:
Removed this snippet and added the java matrix for each job. All workflows look to be running now.